### ISSUE TITLE
`yamllint` is not using config file in `cwd` when being given a `stdin`

### ISSUE THREAD

**Issue by budimanjojo** [adrienverge/yamllint#569]
I have a `.yamllint.yaml` like this to ignore `indentation` for files with `.sops.yaml` extension:
```
---
extends: default
rules:
  indentation:
    spaces: 2
    indent-sequences: consistent
    ignore: |
      *.sops.yaml
```
And this is an example `test.sops.yaml` file:
```
---
hello:
    welcome: home
```

In the same directory, when I run `yamllint test.sops.yaml`, there will be no error from `yamllint` because the indentation is being ignored (which is my desired result). But, if I run `cat test.sops.yaml | yamllint -`, it will return this:
```
stdin
  3:5       warning  wrong indentation: expected 2 but found 4  (indentation) 
```

This is required because `null-ls` which is what I'm using to do diagnostic in `neovim` is using `stdin` to call `yamllint`.
Is there any way to get `yamllint` knowing `cwd` of the command being spawned at? Or maybe add a flag i.e `--stdin-filepath` to let `yamllint` knows where the path of the stdin file? Here's the original issue I reported to `null-ls`: https://github.com/jose-elias-alvarez/null-ls.nvim/issues/1531

**Comment by adrienverge** [comment]
Hello neovim user ✌️

I think the issue title is misleading: yamllint *does* read and use the config file, it's just that data passed as `stdin` doesn't have a filename, hence `ignore` directives won't apply (which makes sense, so a `--stdin-filepath` option would be awkard in my opinion).

Maybe you could achieve what you want using a [rule-disabling directive](https://yamllint.readthedocs.io/en/stable/disable_with_comments.html) *inside* your file, when passed as standard input?

```shell
{echo '# yamllint disable rule:indentation'; cat file.yaml;} | yamllint -
```

**Comment by budimanjojo** [comment]
> I think the issue title is misleading: yamllint _does_ read and use the config file

Feel free to edit the title, I don't know how to word it properly :disappointed: 

> Maybe you could achieve what you want using a [rule-disabling directive](https://yamllint.readthedocs.io/en/stable/disable_with_comments.html) inside your file

Thanks! This works fine and it did solve my problem although I prefer using a project level config file.

Should I close this issue? The solution work but it's not exactly the proposed change.

**Comment by adrienverge** [comment]
If the problem is solved you can close the issue :+1:
